src/server/index.js

```diff
@@ -55,7 +55,7 @@ class Server {
 
 export const registerPlugins = async ({ server, config }) => {
   // Handles static files and proxying
-  const plugins = [require('inert'), require('h2o2')]
+  const plugins = [require('@hapi/inert'), require('@hapi/h2o2')]
```

Comment:
Would it make sense to use the import syntax fully in this file, i.e. getting rid of require entirely?

Reply:
Why do you think that would be better?

Reply:
Consistency I guess. Apart from that I don't see any other benefit (or drawback)

Reply:
We could, but the hapi-require-https module would still need to use require as it's optional. Good spot but will keep it as is for now.